src/Tablebot/Plugin/Help.hs

```diff
@@ -0,0 +1,46 @@
+module Tablebot.Plugin.Help where
+
+import qualified Data.Text as T
+import Data.Text (Text)
+import Text.Megaparsec (choice, (<|>), try, eof, (<?>), chunk)
+import Data.Functor (($>))
+import Tablebot.Plugin.Discord
+import Tablebot.Plugin.Parser
+import Tablebot.Plugin.Types
```

Comment:
A slight suggestion from me would be to specify what you're importing from each of these modules, so it's clearer where specific functions etc come from.